terraform/task-definitions/publisher.json

```diff
@@ -0,0 +1,114 @@
+[
+  {
+    "name": "publisher",
+    "image": "govuk/publisher:serve-assets-in-prod",
+    "essential": true,
+    "environment": [
+      { "name": "ASSET_HOST", "value": "www.gov.uk" },
+      { "name": "BASIC_AUTH_USERNAME", "value": "gds" },
+      { "name": "EMAIL_GROUP_BUSINESS", "value": "test-address@digital.cabinet-office.gov.uk" },
+      { "name": "EMAIL_GROUP_CITIZEN", "value": "test-address@digital.cabinet-office.gov.uk" },
+      { "name": "EMAIL_GROUP_DEV", "value": "test-address@digital.cabinet-office.gov.uk" },
+      { "name": "EMAIL_GROUP_FORCE_PUBLISH_ALERTS", "value": "test-address@digital.cabinet-office.gov.uk" },
+      { "name": "FACT_CHECK_SUBJECT_PREFIX", "value": "dev"},
+      { "name": "FACT_CHECK_USERNAME", "value": "govuk-fact-check-test@digital.cabinet-office.gov.uk"},
+      { "name": "GOVUK_APP_DOMAIN", "value": "test.govuk-internal.digital" },
+      { "name": "GOVUK_APP_NAME", "value": "publisher"},
+      { "name": "GOVUK_APP_TYPE", "value": "rack"},
+      { "name": "GOVUK_STATSD_PREFIX", "value": "fargate" },
+      { "name": "GOVUK_ASSET_ROOT", "value": "https://assets.digital.cabinet-office.gov.uk" },
+      { "name": "GOVUK_GROUP", "value": "deploy"},
+      { "name": "GOVUK_USER", "value": "deploy"},
+      { "name": "GOVUK_WEBSITE_ROOT", "value": "https://www.gov.uk" },
+      { "name": "PLEK_SERVICE_CONTENT_STORE_URI", "value": "https://www.gov.uk/api" },
+      { "name": "PLEK_SERVICE_STATIC_URI", "value": "https://assets.publishing.service.gov.uk" },
+      { "name": "RAILS_ENV", "value": "production" },
+      { "name": "RAILS_SERVE_STATIC_FILES", "value": "true" },
+      { "name": "REDIS_HOST", "value": "pink-backend-redis.0f3erf.ng.0001.euw1.cache.amazonaws.com"},
```

Comment:
Is there a CNAME for this that we could use so that it won't change when you re-create the Elasticache instance? If there isn't yet then don't worry.